feugy/atelier
# feat: uses tool full and short names

### What's in there?

The snapshot files generated by toolshot may contain several expectations: one for each tool in the tool box.
However, only the expectation number (Jest manages it) makes the difference. 
This PR leverages the tool short name to distinguish them.

Are included here:

- [x] feat(ui): sends full and short names over the wire
- [x] feat(toolshot): uses tool short name in snapshot
- [x] refactor(ui): uses new tool.fullName attribute
- [x] test(ui): fixes atelier base url

### How to test?

1. run the test suite, and see the difference in snapshot naming (within the files)
2. run `npm start` in `packages/ui`, browse to http://localhost:3001/atelier,  and check every feature of the UI is still working
